src/models/game-media.model.ts

```diff
@@ -0,0 +1,11 @@
+export interface Media {
+  audios: any[];
+  videos: any[];
```

Comment:
`any` is dangerous, especially as we aspire to be the de-facto reference implementation for all of our consumers 😅

Let's ensure these are strongly-typed.

Reply:
I don't know how, I haven't seen a single game with data in those arrays yet. I could strongly type them if I found any examples, maybe a more recent game or a game published by a big company (like Ubisoft or Sony)?

Reply:
If that's the case, let's use `unknown[]`.